README.md

```diff
@@ -33,7 +26,8 @@ detectors             | detector-specific setups
 ```
 git clone https://github.com/ess-dmsc/essdaq.git
 cd essdaq
-./install.sh
+./install.sh          # for Ubuntu
+./install)centos7.sh  # for CentOS 
```

Comment:
Looks like a typo here with the open-parentheses.
And I thought there was just a single script now that autodetects the OS?

Reply:
Typo fixed